# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: netwerk/protocol/websocket/WebSocketChannel.h
# Commit: 64b4a5c112b5
# Full Hash: 64b4a5c112b5bec70574f3d63a751302bbc5460b
# Author: Marian-Vasile Laza <mlaza@mozilla.com>
# Date: 2022-02-12 09:47:43
# Regressor Bug: 1749786
# File Overlap Count: 1
# Description:
#   Bug 1749786 - lint fix r=fix CLOSED TREE
# ==============================================================================

diff -r 8a0a3e09cf26 -r 64b4a5c112b5 netwerk/protocol/websocket/WebSocketChannel.h
--- a/netwerk/protocol/websocket/WebSocketChannel.h	Fri Feb 11 17:19:56 2022 +0000
+++ b/netwerk/protocol/websocket/WebSocketChannel.h	Fri Feb 11 19:41:54 2022 +0200
@@ -237,7 +237,8 @@
   nsCString mHost;
   nsString mEffectiveURL;
 
-  // Set on MainThread before multithread use, used on IO thread, cleared on IOThread
+  // Set on MainThread before multithread use, used on IO thread, cleared on
+  // IOThread
   nsCOMPtr<nsISocketTransport> mTransport;
   nsCOMPtr<nsIAsyncInputStream> mSocketIn;
   nsCOMPtr<nsIAsyncOutputStream> mSocketOut;
@@ -304,8 +305,10 @@
   Atomic<bool> mIncrementedSessionCount;
   Atomic<bool> mDecrementedSessionCount;
 
-  int32_t mMaxMessageSize;  // set on MainThread in AsyncOpenNative, read on IO thread
-  // Set on IOThread, or on MainThread before mDataStarted.  Used on IO Thread (after mDataStarted)
+  int32_t mMaxMessageSize;  // set on MainThread in AsyncOpenNative, read on IO
+                            // thread
+  // Set on IOThread, or on MainThread before mDataStarted.  Used on IO Thread
+  // (after mDataStarted)
   nsresult mStopOnClose;
   uint16_t mServerCloseCode;     // only used on IO thread
   nsCString mServerCloseReason;  // only used on IO thread
@@ -339,7 +342,7 @@
   uint32_t mHdrOutToSend;
   uint8_t* mHdrOut;
   uint8_t mOutHeader[kCopyBreak + 16]{0};
-  
+
   // Set on MainThread in OnStartRequest (before mDataStarted), used on IO
   // Thread (after mDataStarted), cleared in DoStopSession on IOThread or
   // on MainThread (if mDataStarted == false)
